Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
Did this end up fixing things for you? 

Last night you said it would spit out warnings about duplicated dependencies, but when I compiled, I didn't see anything like that. Expected?